bridgesync/processor.go

```diff
@@ -52,8 +52,18 @@ type Bridge struct {
 	Amount             *big.Int       `meddler:"amount,bigint"`
 	Metadata           []byte         `meddler:"metadata"`
 	DepositCount       uint32         `meddler:"deposit_count"`
+	BlockTimestamp     uint64         `meddler:"block_timestamp"`
+	TxHash             common.Hash    `meddler:"tx_hash,hash"`
+	FromAddress        common.Address `meddler:"from_address,address"`
 }
 
+// BridgeResponse is the representation of a bridge event with additional fields
+type BridgeResponse struct {
+	Bridge
+	BridgeHash common.Hash `meddler:"bridge_hash,hash"`
```

Comment:
If we don't want to persist the `BridgeHash` to the db, we can remove the `meddler` tag. Otherwise, let's rename the field to `Hash` and move it to the `Bridge` struct.

```suggestion
	BridgeHash common.Hash
```